server/routes/userProfileRouter.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const router = express.Router()
+const {
+  profile,
+  getProfile,
+  getAllProfiles
+} = require('../controllers/userProfileController')
+
+router.patch('/:userId', profile)
+router.get('/get_profile/:userId', getProfile)
```

Comment:
The JS naming convention for route names is hyphenated case

Eg: `/save-photo`